### DISCUSSION TITLE
Styling problem and false display of "all day events"

### DISCUSSION THREAD

**Discussion by Dale81** [totaldebug/atomic-calendar-revive#1220] [Support]
Hi everyone!
I am new to HA and this great calender. Now I just updated to the newest version of the calendar and it seems that everything got more narrow. I don't use the compact mode, but still the first event of a day is to narrow to the white line above. Any idea how to fix this or in general how to style the event list in the best way?

Also I a have a problem with all day events. I do not need the info, that they run from 00:00 till the next day same time. The entries come from shared iCal and are set as "all day events". Maybe they get misinterpreted by the calendar? When I add an entry manually into a calendar directly in Home Assistant and set it to all day, it is shown correctly as you can see for "Test" on the 12th.

Would be great to get this sorted out :)

Thank you!
Marc

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/98669240/ca6f1d87-1dda-443f-8f7b-31064cacd1e4)

**Comment by Dale81** [comment]
Hi everyone... just wondering if anyone is here who can help me with my questions.

Thanks!

**Reply by marksie1988** [reply]
Hi,

The styling can be changed with card mod, I did a simple video on how to do this here: https://www.youtube.com/live/-5MKd7LY-oc?si=wXv3gADotUzXNFKf

For the multiday events... This is a little trickier, I may revisit this at some point and see if there is a better way around this for the time being you can't have that any other way.
